src/components/common/line-chart.tsx

```diff
@@ -111,12 +146,11 @@ function LineChartFactory() {
           <HorizontalGridLines tickTotal={3} />
           <LineSeries
             style={{fill: 'none'}}
-            strokeWidth={2}
             color={color}
             data={series}
-            onNearestX={enableChartHover ? onMouseMove : null}
+            onNearestX={enableChartHover ? onMouseMove : undefined}
           />
-          <MarkSeries data={hoveredDP ? [hoveredDP] : []} color={color} size={3} />
+          <MarkSeries data={hoveredDP ? [hoveredDP] : []} color={color} />
```

Comment:
why is `size={3}` removed?

Reply:
The same as previous: according to types from @types/react-vis package MarkSeries doesn't have such property

Reply:
it is used, and also in documentation http://uber.github.io/react-vis/documentation/series-reference/mark-series

the type might be wrong